TESTING.md

```diff
@@ -34,8 +33,8 @@ business logic for each of the custom resources and are used as either mixins
 for the custom resources or contain classes that act as helpers, except with
 explicit namespacing. Some libary files are heavily unit tested, others are definitely
 missing much-needed unit test coverage. The following command syntax assumes
-you've made the ChefDK `rspec` your default by running `chef shell-init bash`
-(https://docs.chef.io/ctl_chef.html#chef-shell-init)
+you've made the Chef Workstation `rspec` your default by running `chef shell-init bash`
```

Comment:
When are we actually making the switch over to `zsh` since Apple pushes it as 100% default now? I know we had a long conversation years ago about this but can't remember if we planned to make a change in the future for it or not.